### PR TITLE
[CALCITE] Added support for CREATE TABLE option FALLBACK

### PR DESCRIPTION
**Pull request by michaelvdow**

### CODE REVIEW

File: babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java [hunk 1]
```diff
@@ -50,18 +51,21 @@
 
   /** Creates a SqlCreateTable. */
   public SqlCreateTable(SqlParserPos pos, boolean replace, SetType setType, Volatility volatility,
```
Comment (dasch-1) [inline babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java:53]: Let's keep the existing constructor and have it pass through a null tableAttributes list to the new constructor.

This allows existing callers (including ones outside of the Calcite codebase) to continue functioning even after this change.

File: babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateAttributeFallback.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    if (no) {
+      writer.keyword("NO");
+    }
+    writer.keyword("FALLBACK");
+    if (protection) {
+      writer.keyword("PROTECTION");
+    }
+  }
+
+  public boolean hasNoKeyword() {
```
[the remark this answers is not in the document]
Comment (michaelvdow) [reply babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateAttributeFallback.java:54]: Sure I can remove them. I was basing the design off of SqlTypeNameSpec and they have getters for their private vars, but I wasn't sure if this class would just be used for unparsing or could used for something later down the line that would need to access these vars

[the remark this answers is not in the document]
Comment (AndrewPochapsky) [reply babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateAttributeFallback.java:54]: In the case of them being needed later we can just add them as the need arises I think.

[the remark this answers is not in the document]
Comment (michaelvdow) [reply babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateAttributeFallback.java:54]: Yeah that sounds good to me, I'll omit them for now

File: babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java (hunk 1 above)
Comment (michaelvdow) [reply babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java:53]: Committed a change for this

**Pull request merged**